.gitignore

```diff
@@ -71,6 +71,14 @@ instance/
 # Sphinx documentation
 docs/_build/
 
+# Sphinx examples rst files which is generated by the template
```

Comment:
can we use `docs/source/quick_start/examples/*.rst`？

Reply:
Because in this path, VOC2012Segmentation.rst, index.rst, get_label_statistics.rst, merge_datasets.rst, update_dataset.rst, move_and_copy.rst are not generated by the template.

Reply:
shall we separate generated rst file and normal rst file into different folder?
otherwise we need to maintain this gitignore list everytime we add a new example.